reflex/utils/hosting.py

```diff
@@ -790,14 +814,17 @@ async def get_logs(
                         row_json["timestamp"] = convert_to_local_time(
                             row_json["timestamp"]
                         )
+                    for k, v in row_json.items():
+                        if v is None:
+                            row_json[k] = str(v)
```

Comment:
generally should avoid modifying the dict during iteration, prefer a comprehension here

```python
row_json_w_str = {
    k: v
    for k, v in row_json.items()
    if v is not None
}
```

Reply:
ah, how about I just create a new one then. I still need the None to be printed. For some reason, the tabulate package is dumb enough to trip over a value being None. Skipping likely causes problem, because it's a table, column numbers need to match.

Reply:
fixed in https://github.com/reflex-dev/reflex/pull/2016/commits/02d95ffbbb4b0544cd7c315204041d151e1ca458